mne_bids/config.py

```diff
@@ -5,7 +5,8 @@
 
 BIDS_VERSION = "1.4.0"
 
-DOI = 'https://doi.org/10.21105/joss.01896'
+DOI = """https://doi.org/10.21105/joss.01896,
+https://doi.org/10.5281/zenodo.3739558"""
```

Comment:
I would not add the zenodo DOI here. It's not indexed on Google Scholar and thus doesn't really help anybody.

From the JOSS DOI, one can easily get to Zenodo by clicking on "Software Archive".

Several DOIs for one thing are always problematic, because people get confused what to cite, and then often cite the wrong thing.